Libraries/pbxbuild/Sources/Phase/File.cpp

```diff
@@ -38,7 +38,7 @@ ResolveBuildFiles(Filesystem const *filesystem, Phase::Environment const &phaseE
             continue;
         }
 
-        std::string fileNameDisambiguator;
+        ext::optional<std::string> fileNameDisambiguator;
```

Comment:
`ext::optional<T>` has an implicit constructor from `T`, so when the `std::string` was passed to the function taking `ext::optional<std::string>`, it was implicitly converted. (Note this commit already landed as #205.)